src/vs/server/node/server.main.ts

```diff
@@ -1014,12 +1014,18 @@ export async function main(options: IServerOptions): Promise<void> {
 		} else if (typeof options.port === 'number') {
 			port = options.port;
 		}
+
 		const host = parsedArgs.host || '0.0.0.0';
+		server.on('error', () => {
+			server.close();
+			process.exit();
+		});
```

Comment:
Would be good to log the error `logService.error(e);`
Also `process.exit(1);` to signal it exited with an error

Reply:
It looks like all the errors get logged even without calling `logService`, so I don't think it will be necessary.

Reply:
(when I log it with `logService` it outputs the error twice)